### DISCUSSION TITLE
Lot of BUH usage

### DISCUSSION THREAD

**Discussion by FlexxFR** [Arnold-n/P1P2MQTT#143] [Q&A]
HI, trying to uderstand why my heatpump is using so much BUH lately.
Is this normal behaviour or is there something to tune?

Example1:

![image](https://github.com/user-attachments/assets/571fe0ac-21c7-49db-8ef1-98bb88ded666)

Example2:
![image](https://github.com/user-attachments/assets/f19fccae-45b3-4a81-aff8-1a55e1051ff7)

First I thought it may have something to do with lower outside temperatures. Could be,. but not always, for example in the 2nd screen it's only for the first heating part, after the DHW runs, heating does not require the BUH, export for what I suspect is a defrost cycle.

And then still....why does the DHW run not require the BUH? 
I know it does require BUH when heating above 53C but that is my limit. Floorwater temp is never higher then 53 right? 

one final picture including defrost and COP.
![image](https://github.com/user-attachments/assets/bc50c080-4c30-4de7-8838-68feb5e8c974)

**Comment by FlexxFR** [comment]
After reading the other threat about defrosting cycles I can confirm these shore BUH spikes match the defrost cycles.
![image](https://github.com/user-attachments/assets/89dae6ca-3c3d-4aaa-88c4-95e5609b612e)

But the main question is about the longer BUH runs during a normal hearing cycle.

**Comment by Arnold-n** [comment]
I think the BUH is used during defrosts as a safety measure only when the return water temperature is below 25C, which is often the case in floor heating systems. This is hard to avoid (you can increase LWT or the WD curve, and sometimes a buffer could help) but I doubt that that is energy-efficient overall.

**Reply by FlexxFR** [reply]
So, these long BUH cycles are to be expected when it's cold?
This was last night. Almost 4 hours continuous use of BUH....that's already 10kWh for the BUH alone (compared to the 11.6 for the HP itself). That does not sound very efficient to me :)

I will check if I can find the return water temp and put it in the plot as well. Still, why does the unit not require BUH for a DHW cycle, for me that is indication enough it still can deliver during these lower temps, right?

![image](https://github.com/user-attachments/assets/52361648-dfa0-49ab-ac95-6e8c7d23dcf9)


LWT = leaving water temp, right... is this determined by that temperature curve? (=WD curve?)
If I understand correctly, you want this value as low as possible to be more efficient, just high enough to keep the house warm. (and maybe prevent BUH usage?)

**Reply by Arnold-n** [reply]
Indeed the LWT setpoint is usually determined by the WD curve (but can be set as absolute setting too). The 4-hour BUH run is likely because the heat pump cannot reach LWT setpoint with the compressor alone (at night COP is lower). If you didn't already, you could lower LWT setpoint at night for example by HA, by a program, or by using 24-hour averaging  of outside temperature ([1-0A]=2, which requires [E-08]=0 on older systems unless you have a separate outside temperature sensor).

**Comment by FlexxFR** [comment]
:) Thanks for the reply Arnold but im trying to follow.

**First reaching LWT setpoint.** 
After some digging I really have some doubt about this setting: Should this be on LWT on the first place?
![image](https://github.com/user-attachments/assets/2d7ef551-372b-41b5-949c-84d9cf91a385)

Note sure if I have the right value here:
sensor.hc_fieldsettings_1_00_a3113_lwt_heating_main_wd_curve_low_amb = -5
sensor.hc_fieldsettings_1_01_a3113_lwt_heating_main_wd_curve_high_amb = 16
sensor.hc_fieldsettings_1_02_a3113_lwt_heating_main_wd_curve_low_amb_lwt = 42
sensor.hc_fieldsettings_1_03_a3113_lwt_heating_main_wd_curve_high_amb_lwt = 30

> So I would assume the LWT setpoint would be somewhere between 30 and 42, right?  
Should it be lowered? And how can I do this via HA? I don;t see any LWT value to write to.


**Last night:**
![image](https://github.com/user-attachments/assets/a4c4b751-efc9-4508-883d-4aed980efd3a)

Leaving water reaches 35C (first "bump") 40C (last bump). The last bumps were without BUH but of course the return water is warmer so this can be explained. Not really sure if these are "normal" figures, 
The outside temp was +/- 1C, even +/- 0 for the second half of this graph.

**This afternoon:**
![image](https://github.com/user-attachments/assets/21d1a8c8-6017-40b3-90e7-8d4e8a26d7d3)

I do not see much difference with the night run.
LWT just reaches 37, outside temperature was between 1 -2 C


Finally, I could not understand the last sentence :) 
These settings has to be set on the bridge directly, right? What would be the benefit of using such a slow average?
and what do those settings mean? With external sensor you mean that with this config another temp value must be used, not the one alterma measures itself? 

Sorry for all the questions :)

**Comment by Arnold-n** [comment]
No problem! RT_LWT can remain on RT, the Daikin will (unless the room is warm enough) warm the water to the LWT setpoint determined by the WD-curve (plus/minus the offset that can be given via the bridge). My current offset is +3:
![Screenshot 2024-11-21 at 23-51-14 HC – Home Assistant](https://github.com/user-attachments/assets/5fa7dac4-d266-4df8-a4e9-6483888ef709)

Based on your WD curve the LWT setpoint (which is visible in HA as LWT_setpoint) is around 39C at 0C outside temperature, and your system cannot reach that even with the BUH until around 2am. Currently the bridge can change the WD curve but not easily from HA. It would require a Daikin restart anyway. It is easier to change the LWT via the offset value above. You can see the current LWT setpoint in HA too:
![Screenshot 2024-11-21 at 23-53-46 HC – Home Assistant](https://github.com/user-attachments/assets/2ef53529-6a0a-4f7c-a60e-d54ff9d02227)

The outside temperature is measured by a sensor near the heat exchanger; this sensor is influenced by the cold of the heat exchanger especially when it collects ice, blocks air flow, and the ice can become around -10C. Without averaging the outside temperature over a longer period this leads to an increased LWT setpoint. Also, if you average the outside temperature over 24 hours you will heat more during the day (higher COP) and less during the night (lower COP), but the temperature differences in your home may become larger.

The Daikin has a separate input for an optional 2nd temperature sensor, which can give  better outside temperature measurement than the sensor near the outside unit. Averaging can be set with [1-0A], but if you do not have the 2nd temperature sensor, and you also do not have a recent system, averaging will not work if [E-08] = 0 which is unfortunately its default setting.

**Comment by FlexxFR** [comment]
thanks again for you explanation! Appreciated.

RT_LWT can remain on RT (=Room Thermostat, right). Check

I can't find the HC_LWT Deviation_Heating like in you example.
Is this the same value? (Recent update maybe?)
![image](https://github.com/user-attachments/assets/055ff13f-d1ce-46e4-90b3-2e7e6fbe7bec)
Moreover, the only entities I can find that contain LWT are in FieldSettings. (I assume those only display settigns, not the actual LWT_setpoint). Maybe I'm missing some values?

Long average: Found the setting in the Daikin menu and I have the possibility to set it to 6 -12 -24 - 72. But you mean, even if it;s set there is a possibility it will not work due to the fact I do not have a 2nd sensor. (I can try anyway, see what happens). System is 3-4 years old. It all floor heating so temp changes are very slow anyway.

But that's all theory, now practice :)
I did change the WD curve yesterday and put the max on 35 (for -10).

![image](https://github.com/user-attachments/assets/f7ec62c8-2759-46ab-87a7-6c3356870260)

BUH is still activated, max temp LWT is now 30 (with BUH enabled), while yesterday afternoon it was at least able to reach 38 without BUH. Okay, outside temp last night was -1C and yesterday afternoon (+1.5). I'm still not convinced the system really requires BUH for such long times. On top of that, a DHW cycle provides much higher LWT temps and do not require BUH.

I did put the system on quiet mode, could this has anything to do with it. (Probably not, as DHW is fine)
After searching in the Daikin menu I did found some other (maybe) relevant settings: (sorry in Dutch)
Aangepaste AWT > Nee
Temp vergrendeling: Continu
Backupverwarming > Evenwichtstemp: 0C

I will try to schedule a DHW cycle around 03:00 next night, in the middle of the long BUH run. Or is it not valid to compare both?

**Comment by Arnold-n** [comment]
You're welcome! Deviation_Heating is not the same as R2Toffset_LWT. Deviation_Heating is used by the Daikin to shift the WD curve up or down, and is a climate dial in HA in subdevice HC_LWT which should be available in control mode "L1" and if your system is not in LWT-Absolute mode. R2Toffset_LWT is only used in the bridge internally to correct the Temperature_R2T_Leaving_Water sensor which often has an offset, giving wrong production calculations. I think you may have to add some subdevices in HA/Settings/Devices&Services/MQTT to your dashboard.

Averaging should always work if you set [E-08] to 0, which may increase standby power of the outside unit, so worth to use a 2nd sensor or try if it is really needed. With floor heating I think 24-hour could be possible without losing too much comfort.

A DHW cycle is not really comparable as the boiler heats up, while the house remains in a semi-stable state.

I think quiet mode is the culprit: in level 3 it almost halves your maximum compressor power.

Nederlands is geen probleem! Aangepaste AWT (RT modulation) is nice to have after the WD curve is set correctly, but confuses observations, so better to keep it off for now.
